### ISSUE TITLE
0.1.7 PyPi wheel release.

### ISSUE THREAD

**Issue by tjhowse** [rpvelloso/Sungrow-Modbus#10]
Could we please get a 0.1.7 build pushed to pypi? That will unblock a modbus4mqtt 0.6.2 release.

Cheers,
tjhowse.

**Comment by rpvelloso** [comment]
@tjhowse I've made an async version of the decorator, will release here soon. I've updated to the latest version of home assistant and since 2024 the sync modbus client does not work anymore. I'm finishing a custom_component for home assistant as well.

**Comment by rpvelloso** [comment]
Yes, I'm using it daily on my home assistant. I've built a component based on the Modbus component.

https://github.com/rpvelloso/Sungrow-Modbus/blob/master/custom_components%2Fsungrowmodbus%2Fsungrow.py

**Comment by tjhowse** [comment]
Sweet, thanks very much! Let me run some more integration tests and I'll let you know shortly.
